qiita_pet/static/js/networkVue.js

```diff
@@ -396,7 +396,8 @@ Vue.component('processing-graph', {
         }
 
         if (data['job_status'] === 'error' && data['job_error'] !== null) {
-          rowsContent.push(['Error message:', data['job_error']]);
+          // based on https://stackoverflow.com/a/14129989
+          rowsContent.push(['Error message:', $('<div>').text(data['job_error']).html()]);
```

Comment:
This fix seems to be related to the problem display error messages (as noted in the issue). Would you mind explaining what's going on here? It seems to me that the incorrectly displayed characters are "\n" not any of the HTML tags, so it's not clear to me how this would help improve the display.

Reply:
Some errors will add a bunch of html to the message, that html gets embedded into the main page, which causes the buttons and other elements to look weird; the solution: make sure the errors are encoded/displayed correctly.

Reply:
Got it, thanks for the explanation!